### ISSUE TITLE
new release

### ISSUE THREAD

**Issue by trebmuh** [sampov2/foo-yc20#9]
Hi @sampov2 ,
I wonder if it would'nt be a good idea to release a new version since the current git master is 79 commits ahead of the 1.3.0 release ?
What do you think ?

**Comment by sampov2** [comment]
It might make sense but unfortunately I do not have an environment to do a proper build. This means that it's difficult for me to test the release, and of course it makes it impossible to build the Windows / OS X releases. Do you happen to know of any solution for this? Perhaps pro bono automated build systems for open source projects?

**Comment by trebmuh** [comment]
No, I don't now automatic building stuff, but I've been seeing a bunch of projects on github using "travis". As a an example : https://github.com/gillesdegottex/fmit
Hope that helps.

**Comment by jpcima** [comment]
> Automatic builds would be great! I'll add you as a collaborator.
>
> It would be especially great if that could be achieved for the Windows and Mac VST's since those builds are especially tricky. But of course the primary goal should be the Linux builds and everything else on top of that is just sugar.

I can probably manage a setup of automatic builds.
I have already a docker base for the cross-compiler builds of Windows and Mac. I would have to complete such a base with GTK deps, Jack and others. For Linux, I can set up the Debian stretch base.

I have to get some familiarity with the build process too, as I had a quick look at existing builds.

1. do we have a storage place, to which is copyable the results of builds by rsync?
2. can you make the CI project and add me? (jpcima on semaphoreci.com)

**Comment by jpcima** [comment]
I've started by setting up the first build.
For build configurations, I'll set up some some reference build images on Docker [here](https://hub.docker.com/r/jpcima/foo-yc20/) (which you can pull for yourself for local builds). I'll have these as `foo-yc20:linux`, `foo-yc20:windows`, `foo-yc20:mac`.

For the deployments, I can set up using my own space, but I'd rather if the projects can have its own.
(because, I'm possibly high already on space quota due to other deployments I also have)

**Comment by trebmuh** [comment]
ping @sampov2

**Comment by trebmuh** [comment]
ping @sampov2
